src/object.js

```diff
@@ -39,7 +40,8 @@ define([
 	 * Creating instances:
 	 *
 	 *    var milou = dog({name: milou});
-	 *    milou.say('hello Tintin'); ;; => 'Woof Woof, hello Tintin'
+	 *    console.log(milou.say('hello Tintin')); ;; => 'Woof Woof, hello Tintin'
```

Comment:
I know @NicolasPetton *loves* tu use `;;` for comments, but it's still `//` in JavaScript :smile: